apps/tunes/views.py

```diff
@@ -0,0 +1,60 @@
+import logging
+
+from django.core.exceptions import SuspiciousOperation
+from rest_framework import generics
+
+from tunes.forms import BrowseSongsForm
+from tunes.serializers import SongSerializer
+from tunes.utils import generate_browse_playlist
+
+
+class BrowseView(generics.ListAPIView):
+    """
+    Return a JSON response of Song records that match a given inout query params.
+    The main thing that should be passed is an `emotion_name`, which denotes the emotion
+    of the songs that should be returned.
+    """
+    serializer_class = SongSerializer
+    logger = logging.getLogger(__name__)
+    default_jitter = .25
+    default_limit = 10
+
+    def __init__(self):
+        self.cleaned_data = {}  # Cleaned GET data for query
+        super(BrowseView, self).__init__()
+
+    def get_queryset(self):
+        user = self.request.user
+
+        emotion = self.cleaned_data['emotion']
+        jitter = self.cleaned_data['jitter']
```

Comment:
Note: Be sure to pull the `limit` parameter here. Follow the same pattern as the jitter: access the key, check if is None: set to default limit.